### ISSUE TITLE
Possible typo in Annex A

### ISSUE THREAD

**Issue by ejossev** [openid/OpenID4VCI#247]
The Annex A, part A1.1.4 (and others) contains this two sentences:

"The following additional parameters are defined for Credential Requests and this Credential format.

credential_definition: REQUIRED when the credential_identifier was not present in the Credential Request. "

I guess the last term shall be "Token Response".

**Comment by jogu** [comment]
I think it's okay, though the tense is wrong - it makes sense if "was not" is replaced with "is not", and that makes it consistent with A1.2.4 / A2.4 / A3.4.

**Comment by ejossev** [comment]
Yes, that is also an option. But IMHO the current wording is just too confusing.

**Comment by Sakurann** [comment]
> credential_definition: REQUIRED when the credential_identifier was not present in the Credential Request.

the sentence is actually correct, but I see where the confusion is coming from. changed to 

> * `credential_definition`: REQUIRED when the `format` parameter is present in the Credential Request.

in https://github.com/openid/OpenID4VCI/pull/243/commits/518de43b65cbe804fd291cd43acb234683d077e2 in PR #243.
